source/index.html

```diff
@@ -33,52 +33,54 @@
+        <!-- angular from npm install to node_modules directory -->
+        <script src="angular-local-storage/dist/angular-local-storage.min.js"></script>
+        <script src="assets/js/lib/x2js-v1.1.5/xml2json.min.js"></script>
 
-        <script src="/assets/js/lib/fileupload/angular-file-upload.min.js"></script>
+        <script src="assets/js/lib/fileupload/angular-file-upload.min.js"></script>
 
-        <script src="/assets/js/ng-app.js"></script>
+        <script src="assets/js/ng-app.js"></script>
 
-        <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
-        <link rel="icon" href="/favicon.ico" type="image/x-icon">
+        <link rel="shortcut icon" href="source/favicon.ico" type="image/x-icon">
+        <link rel="icon" href="source/favicon.ico" type="image/x-icon">
 
         <!-- mvp styles -->
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/bootstrap/bootstrap.min.css" />
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/bootstrap/mvpready-admin.css" />
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/bootstrap/mvpready-flat.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="assets/css/bootstrap/bootstrap.min.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="assets/css/bootstrap/mvpready-admin.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="assets/css/bootstrap/mvpready-flat.css" />
 
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/js/lib/datepicker/css/datepicker.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="assets/js/lib/datepicker/css/datepicker.css" />
 
-        <link href="/assets/js/lib/ng-grid/ui-grid-stable.min.css" rel="stylesheet" />
+        <link rel="stylesheet" type="text/css" href="assets/js/lib/ng-grid/ui-grid-stable.min.css" />
 
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/font-awesome/font-awesome.min.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="assets/css/font-awesome/font-awesome.min.css" />
 
         <!-- Production JavaScript -->
-        <!--<script src="assets/js/dist/profile-editor.min.js"></script>-->
+        <!--<script src="../dist/assets/js/sinopia_profile_editor.min.js"></script>-->
         <!-- Dev JavaScript -->
-        <script src="/assets/js/dist/concat.js"></script>
+        <script src="assets/js/concat.js"></script>
 
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/dist/profile-editor.min.css" />
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/jquery.fileupload.css" />
-        <link rel="stylesheet" type="text/css" media="all" href="/assets/css/jquery.fileupload-ui.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="../dist/assets/css/sinopia_profile_editor.min.css" />
```

Comment:
If you change this href to `href="assets/css/sinopia_profile_editor.min.css`, this stylesheet will load correctly